src/ReaderScreen/components/AudioPlayer/components/AudioSettingsModal.jsx

```diff
@@ -101,7 +86,7 @@ const AudioSettingsModal = ({ isLyricsAvailable }) => {
             </CustomText>
             <Pressable
               onPress={() => handleSpeedChange(audioPlaybackSpeed - 0.1)}
-              disabled={audioPlaybackSpeed < 1.0}
+              disabled={audioPlaybackSpeed <= 0.5}
```

Comment:
Shouldn't this either be `audioPlaybackSpeed < 0.5`, or add <= in the `handleSpeedChange` function.